src/libraries/Microsoft.PowerFx.Connectors/OpenApiHelperFunctions.cs

```diff
@@ -8,12 +8,12 @@
 
 namespace Microsoft.PowerFx.Connectors
 {
-    internal class OpenApiHelperFunctions
+    public class OpenApiHelperFunctions
     {
         private static readonly Regex NotAnXsdNCNameCharRegex = new Regex(@"[^a-zA-Z0-9_-]+", RegexOptions.Compiled);
         private static readonly Regex XsdNCNameStartCharRegex = new Regex(@"^[a-zA-Z_]", RegexOptions.Compiled);
 
-        internal static string NormalizeOperationId(string operationId)
+        public static string NormalizeOperationId(string operationId)
```

Comment:
If it is required, we need to make it public so we also call it to match the function.name.